data/months.json

```diff
@@ -0,0 +1,86 @@
+[
+    {
+        "name": {
+            "en": "January",
+            "fr": "janvier"
+        },
+        "short": "January"
```

Comment:
again the short, wouldn't codes make more sense here? `jan`,`feb`, ext.

Reply:
Same reason as the longer language names, used the real shorts initially but API requires entire English word

Reply:
Just for the reflag. _We (Mohamad) build this API right? If so we should change it._